lib/jsdom/living/xhr/FormData-impl.js

```diff
@@ -170,9 +169,9 @@ function constructTheFormDataSet(form, submitter) {
 }
 
 function isBlob(obj) {
-  return obj !== null && typeof obj === "object" && blobSymbols.buffer in obj;
+  return obj instanceof Blob;
```

Comment:
> That's not true

But it is. I just tested it and [the docs agree.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)